source/operations/install-deploy-manage/deploy-minio-tenant-helm.rst

```diff
@@ -56,16 +56,29 @@ Deploy a MinIO Tenant using Helm Charts
 
 The following procedure deploys a MinIO Tenant using the MinIO Operator Chart Repository.
 This method supports a simplified installation path compared to the :ref:`local chart installation <deploy-tenant-helm-local>`.
-You can modify the Operator deployment after installation.
+
+
+The following procedure uses Helm to deploy a MinIO Tenant using the official MinIO Tenant Chart.
 
 .. important::
 
-   Do not use the ``kubectl krew`` or similar methods to update or manage the MinIO Tenant installation.
-   If you use Helm charts to deploy the Tenant, you must use Helm to manage that deployment.
+   If you use Helm to deploy a MinIO Tenant, you must use Helm to manage or upgrade that deployment.
+   Do not use ``kubectl krew``, Kustomize, or similar methods to manage or upgrade the MinIO Tenant.
```

Comment:
I think keep it because it _was_ around at one point. We want to emphasize that even if someone was using or heard of or read of using `krew`, its no longer recommended.